contact/pubspec.lock

```diff
@@ -611,8 +611,8 @@ packages:
     dependency: "direct main"
     description:
       path: "."
-      ref: main
-      resolved-ref: b1b0d407e5082f9d7afc88ccec8d05bf39ec992f
+      ref: add-smime-status-header-property-for-email
```

Comment:
please bump minor version for `jmap-dart-client` and update it here

Reply:
Please review it at here https://github.com/linagora/jmap-dart-client/pull/95